content/id/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Mengoperasikan klaster etcd untuk Kubernetes
+content_type: task
+---
+
+<!-- overview -->
+
+{{< glossary_definition term_id="etcd" length="all" prepend="etcd adalah ">}}
+
+
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+
+## Prerequisites
+
+* Jalankan etcd sebagai klaster dari anggota ganjil.
+
+* etcd adalah sistem terdistribusi berbasis _leader_. Pastikan _leader_ secara berkala mengirimkan _heartbeat_ dengan tepat waktu ke semua pengikutnya untuk menjaga kestabilan klaster.
+
+* Pastikan tidak terjadi kekurangan sumber daya.
+
+  Kinerja dan stabilitas dari klaster sensitif terhadap jaringan dan _IO disk_. Kekurangan sumber daya apa pun dapat menyebabkan _timeout_ dari _heartbeat_, yang menyebabkan ketidakstabilan klaster. Etcd yang tidak stabil mengindikasikan bahwa tidak ada _leader_ yang terpilih. Dalam keadaan seperti itu, sebuah klaster tidak dapat membuat perubahan apa pun ke kondisi saat ini, yang menyebabkan tidak ada _Pod_ baru yang dapat dijadwalkan.
+
+* Menjaga kestabilan klaster etcd sangat penting untuk stabilitas klaster Kubernetes. Karenanya, jalankan klaster etcd pada mesin khusus atau lingkungan terisolasi untuk [persyaratan sumber daya terjamin](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/hardware.md#hardware-recommendations).
+
+* Versi minimum yang disarankan untuk etcd yang dijalankan dalam lingkungan produksi adalah `3.2.10+`.
+
+## Persyaratan sumber daya
+
+Mengoperasikan etcd dengan sumber daya terbatas hanya cocok untuk tujuan pengujian. Untuk peluncuran dalam lingkungan produksi, diperlukan konfigurasi perangkat keras lanjutan. Sebelum meluncurkan etcd dalam produksi, lihat [dokumentasi referensi persyaratan sumber daya](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/hardware.md#example-hardware-configurations).
+
+## Memulai Klaster etcd
+
+Bagian ini mencakup bagaimana memulai klaster etcd dalam Node tunggal dan Node multipel.
```

Comment:
oh nice one, I just learned that multipel is Indonesian word https://kbbi.kemdikbud.go.id/entri/multipel

Reply:
Thanks, I learned it from @ariscahyadi